brightray/browser/mac/notification_presenter_mac.mm

```diff
@@ -25,6 +25,13 @@
   return nullptr;
 }
 
+void NotificationPresenterMac::RemoveAllNotifications() {
+  NotificationPresenter::RemoveAllNotifications();
+
+  // Remove notifications created by previous processes too
+  [NSUserNotificationCenter.defaultUserNotificationCenter removeAllDeliveredNotifications];
+}
```

Comment:
If I'm understanding this right, this override is here because there can be notifications hanging around not represented in `notifications_`, is that correct?

If so, does it make sense to reimplement `NotificationPresenter::~NotificationPresenter` as a call to `RemoveAllNotifications`?